src/pages/admin/regulatory/forms/CreateRegulatoryForm.js

```diff
@@ -137,8 +137,8 @@ const CreateRegulatoryForm = props => {
                       </Grid>
                       <Grid item >
                         <Button color = "primary"
-                            onClick = {() => {arrayaHlpers.push(regulatoryAttch)}}>
-                              {contacts && contacts.length >0 ? "Add Another Contact" : "Add Contact"}
+                            onClick = {() => {arrayHelpers.push(attachment)}}>
```

Comment:
This is where we push an empty attachment object to the attachments object array